tests/avocado/test_nec-usb-xhci.py

```diff
@@ -19,6 +21,21 @@ class TestNecUsbXhci(test.QemuTest):
     def setUp(self):
         self.vm_dst = None
         usbdevice = os.path.join(self.workdir, 'usb.img')
+
+        self.image = vmimage.get('Fedora')
+
+        virt_customize = utils_path.find_command('virt-customize')
+        process.run('%s --add %s --password %s:password:%s --uninstall %s' %
+                    (virt_customize, self.image, 'root', '1234', 'cloud-init'))
```

Comment:
I don't think this simplifies the tests. I think we should simply rely on users to provide the right image and if we want to be really nice, we should just add another `make` target, or script to download and prepare image using the `vmimage` library.